browser-extension/install.sh

```diff
@@ -2,12 +2,6 @@
 
 npm i -g web-ext
 
-echo -e "\n***** Installing analytics vendor's dependencies *****"
```

Comment:
removed this coz this is already done as part of browser-extention/common  preinstall scripts.

Reply:
This is required as preinstall script only builds the analytics vendor. It doesn't install the dependencies of the module

Reply:
the dependencies of the analytics-vendor module is installed in https://github.com/requestly/requestly/blob/master/install.sh